roles/confluent.ksql/templates/ksql_kerberos_ssl_customcerts.properties.j2

```diff
@@ -0,0 +1,50 @@
+# Maintained by Ansible
+{% include './includes/base_ksql.properties.j2' %}
+
+security.protocol=SASL_SSL
+sasl.mechanism=GSSAPI
+sasl.kerberos.service.name=kafka
+sasl.security.protocol=SASL_SSL
+producer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+consumer.confluent.monitoring.interceptor.security.protocol=SASL_SSL
+
+
+#SSL additions
+ssl.truststore.location=/var/ssl/private/client.truststore.jks
+ssl.truststore.password={{host_truststore_storepass}}
+ssl.keystore.location=/var/ssl/private/client.keystore.jks
+ssl.keystore.password={{host_keystore_storepass}}
+ssl.key.password={{host_cert_password}}
+
+confluent.monitoring.interceptor.ssl.truststore.location=/var/ssl/private/client.truststore.jks
+confluent.monitoring.interceptor.ssl.truststore.password={{host_truststore_storepass}}
+confluent.monitoring.interceptor.ssl.keystore.location=/var/ssl/private/client.keystore.jks
+confluent.monitoring.interceptor.ssl.keystore.password={{host_keystore_storepass}}
+confluent.monitoring.interceptor.ssl.key.password={{host_cert_password}}
+confluent.monitoring.interceptor.security.protocol=SASL_SSL
+
+
+#SASL additions
+sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required \
+   useKeyTab=true \
+   storeKey=true \
+   keyTab="{{ksql.keytab}}/client-{{inventory_hostname}}.keytab" \
+   principal="client/{{inventory_hostname}}@{{realm|upper()}}";
+
+producer.confluent.monitoring.interceptor.sasl.jaas.config=com.sun.security.auth.module.Krb5LoginModule required \
+   useKeyTab=true \
+   storeKey=true \
+   keyTab="{{ksql.keytab}}/client-{{inventory_hostname}}.keytab" \
```

Comment:
Should probably document the naming conventions for the keytabs files to be generated

Reply:
created https://confluentinc.atlassian.net/browse/PLAT-602
to track

Reply:
docs jira has been filed, should get updated shortly.